backend/utils/plugins.py

```diff
@@ -354,6 +354,9 @@ def _single(app: App):
                       response.text[:100])
                 return
 
+            if plugin.uid is None or plugin.uid != uid:
+                record_plugin_usage(uid, plugin.id, UsageHistoryType.transcript_processed_external_integration, memory_id=memory_id)
```

Comment:
any estimation for document inserts / seconds sir ?

Reply:
No sir haven't tested it enough to give estimated numbers

Reply:
do it pls, use our current data.

Reply:
Had a lot of apps enabled and the usage was being recorded for every 2-3 seconds (not fixed all the time, depends on the segments difference), and everytime all the enabled apps get called so the writes per second for a user would be `total number of enabled apps`

For a 15 min YT video during test, the `record_plugin_usage ` was called 234 times in total